src/App.js

```diff
@@ -12,15 +12,20 @@ const App = () => {
 
   useEffect(() => {
     (async () => {
-      const { data } = await axios.get(
-        `https://api.unsplash.com/photos/random?client_id=${process.env.REACT_APP_UNSPLASH_KEY}&orientation=landscape`
-      );
-      const bg = new Image();
-      bg.src = data.urls.raw;
-      bg.onload = () => {
-        setBgImage(data.urls.raw);
-      };
-      setBgImage(data.urls.thumb);
+      try {
+        const { data } = await axios.get(
+          `https://api.unsplash.com/photos/random?client_id=${process.env.REACT_APP_UNSPLASH_KEY}&orientation=landscape`
+        );
+        const bg = new Image();
+        bg.src = data.urls.raw;
+        bg.onload = () => {
+          setBgImage(data.urls.raw);
+          localStorage.setItem("lastBg", data.urls.raw);
+        };
+        setBgImage(data.urls.thumb);
+      } catch (error) {
+        setBgImage(localStorage.getItem("lastBg"));
```

Comment:
if error  occur in  first run so localstorage will not be initialised it will return null so how you are taking care of that?

Reply:
okey got i missed that fallback part:)